src/pubsub.js

```diff
@@ -186,6 +269,10 @@ class BasicPubSub extends Pubsub {
       if (err) {
         return callback(err)
       }
+      // if fallback to floodsub enabled, we need to listen to its protocol
+      if (this._options.fallbackToFloodsub) {
+        this.libp2p.handle('/floodsub/1.0.0', this._onConnection)
```

Comment:
consider pulling out the string into a constant.

Reply:
wait sorry! I will change it, I already have the constant 😅